samples/nrf9160/modem_shell/README.rst

```diff
@@ -559,6 +559,42 @@ Examples
      location get --method cellular
      cloud disconnect
 
+----
+
+Remote control using nRF Cloud
+==============================
+
+Once you have established an MQTT connection to nRF Cloud using the ``cloud`` command, you can use the :guilabel:`Terminal` window in the nRF Cloud portal to execute MoSh commands to the device.
+This feature enables full remote control of the MoSh application running on a device that is connected to cloud.
+MoSh output, such as reponses to commands and other notifications can be echoed to the ``messages`` endpoint and the :guilabel:`Terminal` window of the nRF Cloud portal.
+Use the ``print cloud`` command to enable this behavior.
+The data format of the input data in the :guilabel:`Terminal` window must be JSON.
+
+Examples
+--------
+
+* Establish the connection to nRF Cloud:
+
+  .. code-block:: console
+
+     cloud connect
+
+* To request the device location, enter the following command in the :guilabel:`Terminal` window of the nRF Cloud portal:
+
+   .. code-block:: console
+
+     {"appId":"MODEM_SHELL", "data":"location get --method cellular"}
```

Comment:
it is certainly fine to define this format, but you do not need to since this is only for the MoSH sample.
you could simplify it with something like: `{"mosh_cmd":"location get --method cellular"}`

Reply:
Ok, thanks for pointing out. I was not actually sure if I can come up with new attributes. But I think I'll keep as it is because I don't think one way is clearly better than the other.